libs/langgraph/langgraph/pregel/algo.py

```diff
@@ -328,6 +331,26 @@ def apply_writes(
                         max_version,
                         channels[chan],
                     )
+                    # unavailable channels can't trigger tasks, so don't add them
```

Comment:
And this is necessary here because `update()` can flip the "finished" bit on the deferred barriers and make them no longer available?

Reply:
yea